src/goto-symex/execution_state.cpp

```diff
@@ -843,9 +843,28 @@ void execution_statet::get_expr_globals(
     {
       return;
     }
+
+    bool point_to_global = false;
+    if (symbol->type.is_pointer() && symbol->name != "invalid_object")
+    {
+      expr2tc tmp = expr;
+      cur_state->rename(tmp);
+
+      std::unordered_set<expr2tc, irep2_hash> symbols;
+      get_symbols(tmp, symbols);
```

Comment:
does this return all objects a pointer can point to?

Reply:
`rename` replaces a symbol expression with the last known expression
```
int global;
void func(int *a){*a = 1; } 
int main(){ func(&global); }

```
In this example, the expr for `a` will be replaced by the expr for `global`, than we use `get_symbols()` to get the symbol expr in the result.